pkg/Microsoft.Private.PackageBaseline/packageIndex.json

```diff
@@ -1548,9 +1548,10 @@
         "4.8.0",
         "4.8.1",
         "4.8.2",
-        "4.8.3"
+        "4.8.3",
+        "4.8.4"
       ],
-      "BaselineVersion": "4.8.3",
+      "BaselineVersion": "4.8.4",
```

Comment:
This wasn't changed in the old commit https://github.com/dotnet/corefx/commit/ad726016a345c82123141b63bb8fecfc3c887f86 .

According to the discussion [the old PR](https://github.com/dotnet/corefx/pull/42952/files#r455247565), the baseline wasn't expected to be bumped. @ericstj is it still the case here too? Note that the old discussion was specific to the Microsoft.Windows.Compatibility section.

Reply:
@JRahnama this conversation was resolved without an answer. Did you get a chance to confirm if the change was needed or not?

Reply:
@ericstj @ViktorHofer if you could help confirm this, that would be extremely helpful

Reply:
BaseLineVersion is used to control the version referenced by other packages when their assemblies have a SqlClient reference (even older versions).  It's fine to update that.